docs/architecture.md

```
@@ -105,13 +106,13 @@ Works the same way as the Python bridge — starts a Java process and communicat
 
 ## How Communication Works
 
-### Rust (In-Process)
+### Rust, C, and C++ (In-Process)
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)